policyengine_us/variables/gov/states/mi/tax/income/deductions/mi_standard_deduction.py

```diff
@@ -0,0 +1,83 @@
+from policyengine_us.model_api import *
+
+
+class mi_standard_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Michigan standard deduction"
+    unit = USD
+    definition_period = YEAR
+    documentation = "Michigan standard deduction, consist of standard deduction for age 67-76, retirement and pension benefits for age 62-66 and above 77, and interest, dividends, and capital gains deduction for age above 77."
+    reference = (
+        "https://www.michigan.gov/taxes/iit/retirement-and-pension-benefits/michigan-standard-deduction",
+        "https://www.michigan.gov/taxes/-/media/Project/Websites/taxes/Forms/2022/2022-IIT-Forms/BOOK_MI-1040.pdf?rev=86a928564e3f42449c531309673f5da7&hash=7147C48E7C9B1B8171B72DC34A66642A",
+        "https://www.michigan.gov/taxes/iit/retirement-and-pension-benefits",
+    )
+    defined_for = StateCode.MI
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.mi.tax.income.deductions.standard
+        # Core deduction based on filing status.
+        filing_status = tax_unit("filing_status", period)
+
+        age_older = max_(
+            tax_unit("age_head", period), tax_unit("age_spouse", period)
+        )
+        # Michigan Standard Deduction
+        age_min_threshold = p.min_age
```

Comment:
In general, don't create new variables with names longer than the thing they're aliasing. Also always use `&` instead of `and` for vectorization.
```suggestion
        aged_count = (age_older >= p.min_age) & (age_older <= p.max_age)
```